src/commands/new.ts

```diff
@@ -203,6 +205,11 @@ module.exports = {
     await system.run(yarnOrNPM)
     spinner.succeed(`${yarnOrNPM} complete`)
 
+    // run install cocoapods
+    if (isMac === true) {
+      await installCocoapods()
+    }
```

Comment:
Rather than running it here, I think it makes sense to provide it to the boilerplate and have the boilerplate itself run this command.

For example, in Bowser we already do something similar, but it would be good to include the functionality in Ignite CLI so each boilerplate doesn't have to reimplement it.

During install, we create a `postInstall` hook to run a bunch of stuff, including `pod install`: https://github.com/infinitered/ignite-bowser/blob/cfa6d3f58716977793075dd6789dc90f6ac8537b/src/boilerplate.ts#L254

Running yarn triggers pod install: https://github.com/infinitered/ignite-bowser/blob/cfa6d3f58716977793075dd6789dc90f6ac8537b/src/boilerplate.ts#L264-L267

postInstall hook: https://github.com/infinitered/ignite-bowser/blob/cfa6d3f58716977793075dd6789dc90f6ac8537b/boilerplate/bin/postInstall#L24

Reply:
I'm not entirely sure what the right process might look like here. A postinstall hook for running `pod install` is a good general solution I think.

Reply:
Thanks for taking a time to response :)

I think i have missed that, and also i did not put platform `web` in consideration. i'll look at `Bowser` boilerplate again

You could close this pr